README.md

```diff
@@ -81,7 +81,27 @@ route -p add 192.168.91.1 mask 255.255.255.255 192.168.0.100
 
 #### Powerwall 3 Mode (beta)
 
-If you have access to the Powerwall Gateway (see local mode Extended Device Vitals Metrics note above), you can select option 4 to activate Powerwall 3 mode. All data will be pulled from the local Gateway TEDAPI endpoint. 
+If you have access to the Powerwall Gateway (see local mode Extended Device Vitals Metrics note above), you can select option 4 to activate Powerwall 3 mode. All data will be pulled from the local Gateway TEDAPI endpoint.  
+
+Before running ./setup.sh  Perform the following steps:
+* copy of pypowerwall.env.sample and rename it pypowerwall.env
+* Update file contents to include PW_GW_PWD paratmeter the value being your Tesla Powerwall3 wifi password.
+* Update PW_EMAIL and PW_PASSWORD to be empty.
+* Save and run ./setups.sh
```

Comment:
First, THANK YOU for opening a PR.

Now, as to this - you should not need to do this. The first thing setup.sh does it copy that file and creates the PW_GW_PWD entry if you select PW3 mode, and then create the empty PW_EMAIL and PW_PASSWORD fields. 

I think setup.sh failed for you because the `curl` command used to detect the gateway failed. I need to add more error handling to the setup.sh that basically says, "I didn't find the Powerwall GW, do you want to retry?" If you say no, it should continue as if the GW works and set up all the above you list out.

Having said that, I do think this data would be good to include in the Troubleshooting section for PW3, including the working evn file example below.

Reply:
The logic here is a bit backward. The retry curl will need to be in the else branch (return = 0 is consider successful run in bash, which I understand seems backward).  However, my issue with doing this is that if your host is having trouble running `curl` against the gateway, then pypowerwall will as well. I would rather detect that issue early so that the system owner can investigate and fix it.

I made some changes to your PR to have the `setup.sh` provide more information to the user. It also occurs to me that PW3 owners may select option 1 (local mode) and not know that the PW3 can't support the `hybrid` data. The script now asks the user if they have a Powerwall 3 if it detects the GW and will erase PW_PASSWORD and PW_EMAIL to ensure pypowerwall comes up in `full` mode.